content/warnings/unknown-prop.md

```diff
@@ -9,7 +9,7 @@ Hay un par de razones probables por las que podría aparecer esta advertencia:
 
 1. ¿Está utilizando `{...this.props}` o `cloneElement(element, this.props)`? Su componente está transfiriendo sus props directamente a un elemento hijo (por ejemplo, [transfiriendo props](/docs/transferring-props.html)). Al transferir props a un componente hijo, debe asegurarse de que no está enviando accidentalmente props cuya intención fue ser interpretado por el componente padre.
 
-2. Está utilizando un atributo DOM no estándar en un nodo DOM nativo, tal vez para representar datos personalizados. Si está tratando de adjuntar datos personalizados a un elemento DOM estándar, considere usar un atributo de datos personalizados como se describe [en MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes ).
+2. Está utilizando un atributo DOM no estándar en un nodo DOM nativo, tal vez para representar datos personalizados. Si está tratando de adjuntar datos personalizados a un elemento DOM estándar, considere usar un atributo de datos personalizados como se describe [en MDN](https://developer.mozilla.org/es/docs/Learn/HTML/como/Usando_atributos_de_datos).
```

Comment:
Hey @jvelezpo, please, change the wording from `usted` to `tú`.

You can take a look at our [styleguide](https://github.com/reactjs/es.reactjs.org/blob/master/TRANSLATION.md#style-guide) for further reference.

Reply:
hey @alejandronanez i dont see any `usted` in these files that i translated, i am aware of the change `usted` for `tú` that was proposed by RamirezAlex (we work at the same co working space 😎), but again i dont see any usted in my files

Reply:
Oh sorry, I wasn't clear enough, my bad!

You're not literally using `usted` here, but the sentence does, it should be

```
2. Estás utilizando un atributo DOM no estándar en un nodo DOM nativo, tal vez para representar datos personalizados. Si estás tratando de adjuntar datos personalizados a un elemento DOM estándar, considera usar un atributo de datos personalizados como se describe [en MDN](https://developer.mozilla.org/es/docs/Learn/HTML/como/Usando_atributos_de_datos)
```

Same goes for item # 1, 3 and 4 - But that's out of the scope of this PR, so feel free to skip those :)